### ISSUE TITLE
Kimchi/Wok Requiring Internet Access?

### ISSUE THREAD

**Issue by tbarry813** [kimchi-project/kimchi#1271]
We have an environment that has 0 internet access in which we are trying to utilize Kimchi/Wok. We are able to access the login screen but the login does not work. The fields clear and does not successfully log in. From a developer mode in the browser, it appears to be looking for style sheets and scripts from cdn.datatables.net. Is there a workaround for this and is there a way to be successful with this deployment without internet access?

**Comment by alinefm** [comment]
Hi @tbarry813 

Wok and Kimchi started using CDN after 2.5 release which means you can use Kimchi and Wok 2.5.

Otherwise, you need to download the CDN code by yourself, serve it in a local web server and adjust all the Wok and Kimchi web pages to point to the local files.

**Comment by tbarry813** [comment]
We are on version 2.5 which references CDN in the login.html.tmpl and wok-ui.html.tmpl files. We were able to download all of the css files, js files, and the font awesome package (4.7.0). We placed them in the root CSS folder for Wok and then modified the login.html.tmpl and wok-ui.html.tmpl files to point to the local CSS/JS files. That resolved our issue. We did try upgrading to the latest version after the switch to python3 but had many issues with dependencies. We reverted back to 2.5.
